src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -50,6 +49,16 @@ export const CLOSED_STATUSES = [
   STATUS_TYPES.otherClose
 ];
 
+// Action Types & Availability statuses
+// Note: excludes FETCH_APPEALS_SUCCESS / UNAVAILABLE because there are defined in actions
+// and used in v1 as well
```

Comment:
Ugh. This is getting messy, isn't it?

Reply:
Yes, to all the above comments. This needs to be broken out into own reducers for v2, but I think that should be in its own PR.

Reply:
Agreed.